data/blog/developing-advancements-in-onchain-privacy.mdx

```diff
@@ -0,0 +1,224 @@
+batch transactions, Payjoin Dev Kit reduces blockchain fingerprinting and
+strengthens privacy and security for all.
+
+### Coinswap
+
+With financial surveillance becoming more prevalent and intrusive, tools that
+protect privacy have become increasingly important for preserving financial
+sovereignty. [Coinswap] has been actively developing a solution to address this
+challenge—a decentralized atomic swap protocol that enables users to swap
+bitcoin UTXOs, breaking direct ownership links on-chain and disrupting
+chain-analysis heuristics.
+
+Originally proposed by [Greg
+Maxwell](https://bitcointalk.org/index.php?topic=321228.0) in 2013 and later
+developed into a proof-of-concept by [Chris Belcher], the protocol is now
+actively advancing through independent contributors, including developers from
+[Citadel-Tech](https://github.com/citadel-tech), with funding from OpenSats.
+
+Using a maker-taker model, swap service providers (makers) facilitate swaps,
+while swap users (takers) rely on applications to automatically coordinate
+multi-hop transactions for enhanced privacy. By passing bitcoin through multiple
+addresses, these transactions add uncertainty for trackers by obscuring the true
+sender and recipient. This blending of privacy-enhanced transactions with
+standard bitcoin transactions makes it significantly harder for nefarious actors
+to exploit financial surveillance, construct misleading transaction narratives,
+or leverage data for unjust financial profiling.
+
+Since receiving a grant from OpenSats in October last year, development has
+focused on refining the smart-client (taker's application), dumb-server (maker's
+server) model, and integrating [fidelity
+bonds](https://jamdocs.org/glossary/#fidelity-bond) to resist [Sybil
+attacks](https://en.wikipedia.org/wiki/Sybil_attack). Despite the protocol's
+complexity, Coinswap prioritizes user-friendly workflows to make financial
+privacy accessible.
+
+The recent beta release of
+[v0.1.0](https://github.com/citadel-tech/coinswap/milestone/1) marks a major
+milestone towards making Coinswap use more prevalent. This testnet release aims
+to demonstrate how swaps can function in real-world conditions without revealing
+which participants exchanged UTXOs.
+
+Looking ahead, the project plans to implement Taproot outputs, MuSig2 signatures
+for smaller and more private transactions, and decentralized DNS discovery to
+enable permissionless access to swap servers. The project is also exploring
+cross-chain swaps with UTXO-based sidechains like Liquid.
+
+By evolving alongside Bitcoin's broader ecosystem, Coinswap ensures that
+financial privacy remains practical and widely available, empowering individuals
+and businesses to transact freely without fear of surveillance.
+
+### Silent Payments
+
+Developed by OpenSats
+[LTS](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors)
+grantee [Josi Baker](/blog/josi-baker-receives-opensats-lts-grant), Silent
```

Comment:
Will verify with him in person tomorrow, but I thought his first name was Josie.

```suggestion
grantee [Josie Baker](/blog/josi-baker-receives-opensats-lts-grant), Silent
```

Reply:
Good catch, thank you. (We went with Josi [in the past](https://opensats.org/blog/josi-baker-receives-opensats-lts-grant), but I just noticed that he put "josie" in his [GitHub profile](https://github.com/josibake).)

Reply:
Confirmed with him: Josie

Reply:
Thanks for updating! Would suggest updating s/Josi/Josie/ Baker in the blog post this article links to: https://opensats.org/blog/josi-baker-receives-opensats-lts-grant